package/test_requirements.txt

```diff
@@ -1,3 +1,3 @@
 nose
 mock
-unittest2
+unittest2
```

Comment:
what the change here?

Reply:
nothing, just changed this file several times during work and then rolled back, not sure why it shows a change.